src/transaction/atomic_replication_helper.cpp

```diff
@@ -179,11 +235,14 @@ namespace cublog
 	rcv.pgptr = m_watcher.pgptr;
       }
 
+    redo_context.m_reader.advance_when_does_not_fit (sizeof (T));
+    const log_rv_redo_rec_info<T> record_info (m_record_lsa, rectype,
+	redo_context.m_reader.reinterpret_copy_and_add_align<T> ());
     if (log_rv_fix_page_and_check_redo_is_needed (thread_p, m_vpid, rcv.pgptr, record_info.m_start_lsa,
```

Comment:
From the PR description, I was expecting a split in phases : i) all pages in sequence are fixed, ii) all records are applied iii) all pages are unfixed
This function seems to fix  page (log_rv_fix_page_and_check_redo_is_needed)  and then apply (log_rv_redo_record_sync_apply).

Reply:
My mistake, I called the wrong function, changed now